src/ontology/edits/oeo-physical.omn

```diff
@@ -8846,15 +8889,19 @@ Class: OEO_00140133
 
     Annotations: 
         OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
-        <http://purl.obolibrary.org/obo/IAO_0000115> "RE-share is a process attribute that indicates the fraction of renewable energy related to the total energy of an energy generation or consumption process.",
-        <http://purl.obolibrary.org/obo/IAO_0000118> "renewables share",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A renewable energy share is a process attribute that indicates the fraction of renewable energy related to the total energy of an energy generation or consumption process.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "renewables share
+RE share",
```

Comment:
If we have multiple alternative terms, we usually have for each alternative term a separate annotation and do not list them within a single annotation.